spec/helpers/post_peep.rb

```diff
@@ -0,0 +1,9 @@
+module PostPeepHelpers
+
+  def post(message)
```

Comment:
Unused method argument - `message`. If it's necessary, use `_` or `_message` as an argument name to indicate that it won't be used. You can also write as `post(*)` if you want the method to accept any arguments but don't care about them.